docs/user_manual/source/control_status_registers.rst

```diff
@@ -909,7 +988,11 @@ Detailed:
 +==========+===========+=================================================================================================+
 | 31:28    | RO (0x4)  | **xdebugver:** returns 4 - External debug support exists as it is described in this document.   |
```

Comment:
it is described in this document -> should point to RISC-V debug spec instead

Reply:
Done